lib/DynamoAttributesHelper.js

```diff
@@ -1,99 +1,129 @@
+      }
 
-            var params = {
-                Key: {
-                    userId: userId
-                },
-                TableName: table,
-                ConsistentRead: true
-            };
+      const params = {
+        Key: {
+          userId,
+        },
+        TableName: table,
+        ConsistentRead: true,
+      };
 
-            doc.get(params, function(err, data){
-                if(err) {
-                    console.log('get error: ' + JSON.stringify(err, null, 4));
+      doc.get(params, (err, data) => {
+        if (err) {
+          console.log(`get error: ${JSON.stringify(err, null, 4)}`);
 
-                    if(err.code === 'ResourceNotFoundException') {
-                        var dynamoClient = new aws.DynamoDB();
-                        newTableParams['TableName'] = table;
-                        dynamoClient.createTable(newTableParams, function (err, data) {
-                            if(err) {
-                                console.log('Error creating table: ' + JSON.stringify(err, null, 4));
-                            }
-                            console.log('Creating table ' + table + ':\n' + JSON.stringify(data));
-                            callback(err, {});
-                        });
-                    } else {
-                        callback(err, null);
-                    }
-                } else {
-                    if(isEmptyObject(data)) {
-                        callback(null, {});
-                    } else {
-                        callback(null, data.Item['mapAttr']);
-                    }
-                }
+          if (err.code === 'ResourceNotFoundException') {
+            const dynamoClient = new aws.DynamoDB();
+            newTableParams.TableName = table;
+            dynamoClient.createTable(newTableParams, (createErr, createData) => {
+              if (createErr) {
+                console.log(`Error creating table: ${JSON.stringify(createErr, null, 4)}`);
+              }
+              console.log(`'Creating table ${table}:\n ${JSON.stringify(createData)}`);
+              callback(createErr, {});
             });
-        },
+          } else {
+            callback(err, null);
+          }
+        } else if (isEmptyObject(data)) {
+          callback(null, {});
+        } else {
+          callback(null, data.Item.mapAttr);
+        }
+      });
+    },
 
-        set: function(table, userId, data, callback) {
-            if(!table) {
-                callback('DynamoDB Table name is not set.', null);
-            }
+    /**
+     * set - Store user attributes in DynamoDB table.
+     *
+     * @param  {string} table         Name of table to store attributes
+     * @param  {string} userId        User ID
+     * @param  {object} data          Object containing attributes to store
+     * @param  {dbCallback} callback  Callback to handle data or error
+     * @return {undefined}
```

Comment:
I'm not an expert on JSDoc, but I've always included it because the ESLint valid-jsdoc rule requires every function to have an @return. I'm perfectly happy to change that though.